lisa/trace.py

```diff
@@ -588,6 +588,19 @@ def show(self):
             return webbrowser.open(self.ftrace.trace_path)
         self.get_logger().warning('No trace data available')
 
+    def cpus_of_tasks(self, tasks):
+        """
+        Return the list of CPUs where the ``tasks`` executed.
+
+        :param tasks: Task names or PIDs to look for.
+        :type tasks: int or str
```

Comment:
That should be a list, no?
```suggestion
        :type tasks: list(int or str)
```